spec/requests/proceeding_types_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+require 'json_expressions/rspec'
+
+RSpec.describe 'Proceeding types' do
+  describe 'GET proceeding_types' do
+    let!(:proceeding_type) do
+      ProceedingType.create(code: 'PH0001',
+                            ccms_code: 'PBM23',
+                            ccms_category_law: 'Family',
+                            ccms_category_law_code: 'MAT',
+                            ccms_matter: 'Public Law - Family',
+                            ccms_matter_code: 'KPBLB',
+                            meaning: 'Application for a care order',
+                            description: 'to be represented on an application for a care order.')
+    end
+
+    it 'returns a proceeding types' do
+      get '/proceeding_types'
```

Comment:
**[suggestion/not a blocker/does not enforce code changes]** 💭 In cases where we returning a list of values, I find it more _accurate_ to at least expected two element to be returned.